migrations/challenges/sync-all-challenges.js

```diff
@@ -0,0 +1,45 @@
+import Bluebird from 'Bluebird';
+
+import { model as Challenges } from '../../website/server/models/challenge';
+import { model as User } from '../../website/server/models/user';
+
+async function syncChallengeToMembers (challenges) {
+  challenges.forEach(async function (challenge) {
+    let users = await User.find({challenges: challenge._id}).exec();
+
+    users.forEach(async function (user) {

```

Comment:
forEach doesn't work well with async because it has no return value, so nothing here will be waited for. You should transform this code to 

```
    await users.map(function () {
        .... 
        return Bluebird.all(promises);
    })
```